packages/api/src/command/medical/patient/get-consolidate-data.ts

```diff
@@ -0,0 +1,202 @@
+import { OperationOutcomeError } from "@medplum/core";
+import {
+  Bundle,
+  BundleEntry,
+  ExtractResource,
+  OperationOutcomeIssue,
+  Resource,
+  ResourceType,
+} from "@medplum/fhirtypes";
+import { intersection } from "lodash";
+import { makeFhirApi } from "../../../external/fhir/api/api-factory";
+import {
+  isoDateRangeToFHIRDateQuery,
+  resourceSupportsDateQuery,
+} from "../../../external/fhir/shared";
+import { capture } from "../../../shared/notifications";
+import { Util } from "../../../shared/util";
+import { getPatientOrFail } from "./get-patient";
+
+function fullDateQueryForResource(fullDateQuery: string, resource: ResourceType): string {
+  return resourceSupportsDateQuery(resource) ? fullDateQuery : "";
+}
+
+export async function getConsolidatedPatientData({
```

Comment:
ditto, renaming but original is left-over and Git not recognizing rename

Reply:
This was marked as completed but I still see the original file in addition to the new one. Maybe forgot to push changes to remote?

Reply:
Weird the consolidate-data.ts shouldnt be in here anymore. I dont see it in the PR

Reply:
GH having issues with PRs 🙄